setup.py

```diff
@@ -18,7 +18,8 @@ def get_all_scripts(dirname):
         'scipy',
         # Installation from anaconda is recommended for PyTorch
         # 'torch==0.4.1',
-        'chainer==5.0.0',
+        #  need to match to Makefile cupy==4.3.0
```

Comment:
Probably I misunderstood something, but does this implementation depend on cupy? (it should not be, right?)

Reply:
You are right. This is for unittest. I found chainer 5.0.0 does not accept cupy 4.3.0 arrays in the unittest. I can revert this change because I do not matter if the unrelated unittest fails.